### DISCUSSION TITLE
Run PK-Sim on Linux via wine

### DISCUSSION THREAD

**Discussion by janakrw** [Open-Systems-Pharmacology/Forum#1686] [Questions & Problems]
Would like to know if  anybody tried to run PK-Sim  on Linux via wine? I tried but it gave a lot of error messages mainly due to ,NET framework things. Thanks.

**Comment by teutonicod** [comment]
Hi, 
I tried to install PK-Sim on Wine but encounter several issues. I tried Crossover, which is a commercial (and easier to use) version of Wine. I used the demo version for some tests.
https://www.codeweavers.com/crossover

I managed to install PK-Sim and successfully run the installation validator, but the GUI could not be open correctly. I think it was related to DirectX components missing if I recall correctly. I made this test quite some time ago on Ubuntu, OpenSuSe and Manjaro with similar outcome. 

Hope this helps! Good luck and please, let us know if you manage to get a fully working installation of PK-Sim!

Best,

Donato
